src/beautifiers/puppet-fix.coffee

```diff
@@ -8,21 +8,28 @@ module.exports = class PuppetFix extends Beautifier
   # this is what displays as your Default Beautifier in Language Config
   name: "puppet-lint"
   link: "http://puppet-lint.com/"
-  isPreInstalled: false
 
   options: {
     Puppet: true
```

Comment:
Could you change this to `false` -- indicating no options are supported.

I know this is out of scope for your PR however I hope you can make this quick change now that I notice it 😃 . Thank you!

Reply:
No wukkas